.github/workflows/ci.yml

```diff
@@ -61,9 +61,8 @@ jobs:
       - name: Run circom tests
         run: nix develop -c make test-circom-circuits
 
-      # Disable as slot should be added to the contract update function
-      # - name: Run Verifier in Cosmos - Relayer test
-      #   run: nix develop -c yarn test './tests/cosmosLightClient/test-verifier-in-cosmos-relay.ts'
+      - name: Run Verifier in Cosmos - Relayer test
+        run: nix develop -c yarn test './tests/cosmosLightClient/test-verifier-in-cosmos-relay.ts'
 
-      # - name: Run Verifier in EOS test
-      #   run: nix develop -c yarn test './tests/eosLightClient/test-verifier-in-EOS.ts'
+      - name: Run Verifier in EOS test
+        run: nix develop -c yarn test './tests/eosLightClient/test-verifier-in-EOS.ts'
```

Comment:
This must remain as comments since we don't have solution for the EOS tests in the CI